dev/tasks/conda-recipes/clean.py

```diff
@@ -0,0 +1,66 @@
+from distutils.version import LooseVersion
+from subprocess import check_output, check_call
+from typing import List
+
+import json
+import os
+import pandas as pd
+import sys
+
+
+VERSIONS_TO_KEEP = 5
+PLATFORMS = ["linux-64", "osx-64", "win-64"]
+PACKAGES = ["pyarrow", "arrow-cpp"]
+
+
+def packages_to_delete(package_name: str, platform: str) -> List[str]:
+    env = os.environ.copy()
+    env["CONDA_SUBDIR"] = platform
+    pkgs_json = check_output(
+        [
+            "mamba",
```

Comment:
Is "mamba" really required for this to work?

Reply:
It would probably also work with `conda search` with a bit higher runtime.